src/js/modules/index.old.js

```diff
@@ -0,0 +1,161 @@
+/* /* eslint-disable func-names
```

Comment:
You ought to remove commented out code:
It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history 

see this [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png)

Please remove

Reply:
Deleted